src/olympia/reviewers/views.py

```diff
@@ -872,6 +893,12 @@ def review(request, addon, channel=None):
         verdict_info = summary.calculate_verdict(pretty=True)
         auto_approval_info[a_version.pk] = verdict_info
 
+    # We want to notify the reviewer if there are versions needing extra
+    # attention that are not present in the versions history (which is
+    # paginated).
+    versions_flagged_by_scanners = versions_qs.filter(
```

Comment:
`additional_versions_flagged_by_scanners` ?  It's a count that doesn't include versions shown on the current page.

Reply:
I wanted to have that originally but it broke 80 characters limit when included in context below (`additional_versions_flagged_by_scanners=additional_versions_flagged_by_scanners`) so I didn't bother.